spec.bs

```diff
@@ -967,8 +966,10 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|:
   1. Let |sellerSignals| be the result of [=evaluating a reporting script=] with
      |sellerReportingScript|, "`reportResult`", and
      « |config|'s [=auction config/auction signals=], |browserSignals| ».
-  1. TODO: Set URLs passed to sendReportTo() into the FencedFrameConfig so they can be fetched when
-    ad renders.
+  1. Let |reportUrl| be [=this=]'s [=relevant global object=]'s

```

Comment:
The "[=this=]'s [=relevant global object=]" thing only works *inside* callbacks like setBid/setPriority/sendReportTo, not outside in the caller of [=evaluating a reporting script=].  We need to make a few changes:
1. [=evaluating a reporting script=] needs to return not only |sellerSignals| but other stuff, like the |reportUrl|.
2. [=evaluating a reporting script=] gets the |reportUrl| to return from |global|'s [=InterestGroupReportingScriptRunnerGlobalScope/report url=]

Reply:
good catch.